pep-9999.rst

```diff
@@ -0,0 +1,501 @@
+PEP: 9999
+Title: tomllib: Support for parsing TOML in the Standard Library
+Author: Taneli Hukkinen, Shantanu Jain <hauntsaninja at gmail.com>
```

Comment:
The first author should be Shantanu Jain, assuming it reflects workload on this PEP document. If on the other hand we also take into account things like authoring the library and integrating it to the stdlib, then I'm happy with the current order.

Reply:
That's something for the two of you as the PEP authors to decide, I don't think there's any strong norm.